docs/content/drafts/Button2.mdx

```diff
@@ -65,6 +65,20 @@ The `invisible` variant of `Button` indicates that the action is a low priority
 </>
 ```
 
+### Selected button
+
+`Button` can be set to in a "selected" state in two ways. You can use a `selected` class or set `aria-selected ="true"`.
```

Comment:
Is it worth adding a `selected` prop that handles both styling and setting aria attributes? May make consumer-side implementation easier. Also, I don't usually ask for users to set classes elsewhere to enable features, so could this be confusing?

Reply:
Same! this was the exact confusion I was trying to discuss on slack as well. prop or class? Class is just not how we'd do this in react

Reply:
Should it be`aria-pressed` instead? 🤔 

> The aria-selected attribute indicates the current "selected" state for gridcell, option, row and tab roles.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected

> The aria-pressed attribute indicates the current "pressed" state of a toggle button.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed

cc @alliethu - what do you think the correct ARIA attribute might be?

Reply:
This came up today in the Primer patterns working session. We decided this should be purely visual and not set any ARIA attributes.

In the case where a button controls a dropdown, it would need to be passed `aria-haspopup` and `aria-expanded` attributes. These attributes would be passed by the component that renders the dropdown button and menu.

cc @langermank

Reply:
Two questions here
- How do we set the pressed styles in this case?
- How do we support the toggles in the tabs use case in the issue https://github.com/primer/react/issues/1783 ?

I'm thinking we need to implement `aria-pressed` styles and scrap the `selected` prop.

Reply:
The two use cases here:

1. Sounds like #1783 is using Button as a "toggle button", which isn't just a visual update but also a semantic change? 

From MDN (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed): 

> Adding aria-pressed to an element with a role of button turns the button into a toggle button. The aria-pressed attribute is only relevant for toggle buttons. It represents the button's current "pressed" state.

It looks like it makes sense to add `aria-pressed` for toggle button. cc @alliethu for confirmation

2. For buttons that open a Menu, we add the aria attributes `aria-haspopup="true" aria-expanded="true"`, _we can_ modify button styles if those are present OR to keep the two concerns independent, we can pass a `pressed` prop or className to the Button from the Menu components.

Reply:
Got it, so @mperrotti @siddharthkp I'm adding a `pressed` prop to the component and we can handle the css based on that. This will address both the above use cases.